doc/functions/enum.rst

```diff
@@ -13,10 +13,10 @@
     {{ enum('App\\MyEnum').SomeCase.value }}
 
     {# get all cases of an enum #}
-    {% enum('App\\MyEnum').cases() %}
+    {% enum('App\\MyEnum').cases %}
 
     {# call any methods of the enum class #}
-    {% enum('App\\MyEnum').someMethod() %}
+    {% enum('App\\MyEnum').someMethod %}
```

Comment:
I would keep this one as a method might have some arguments (maybe we should add some here).